README.md

```diff
@@ -1,44 +1,65 @@
 ## Introduction
 
-A standalone uniform zkpool prover build upon zkp circuit
+This is a universal modular prover that is built upon various zero-knowledge proof (ZKP) provers. 
+It is used to connect with ZKPool in order to obtain proving tasks and generate proofs.
 
+### Supported ZKP projects
+1. Taiko A3
```

Comment:
Are we sure it supports A3 now? Ignore this comment if you have already checked so